scripts/pg-drop

```diff
@@ -0,0 +1,233 @@
+            counter_data = self.counters_db.get(self.counters_db.COUNTERS_DB, full_table_id, counter_name)
+            if counter_data is None:
+                fields[pos] = STATUS_NA
+            elif fields[pos] != STATUS_NA:
+                fields[pos] = str(int(counter_data) -  old_collected_data)
+        return fields
+
+    def print_all_stat(self, table_prefix, key):
+        table = []
+        type = self.pg_drop_types[key]
+        self.build_header(type)
+        # Get stat for each port
+        for port in natsorted(self.counter_port_name_map):
+            row_data = list()
+            data = self.get_counters(table_prefix, type["obj_map"][port], type["idx_func"], type["counter_name"])
+            row_data.append(port)
+            row_data.extend(data)
+            table.append(tuple(row_data))
+
+        print(type["message"])
+        print(tabulate(table, self.header_list, tablefmt='simple', stralign='right'))
+
+
+    def get_counts(self, counters, oid):
+            """
+                Get the pg drop counts for an individual counter.
+            """
+            counts = {}
+            table_id = COUNTER_TABLE_PREFIX + oid
+            for counter in counters:
+                counter_data = self.counters_db.get(self.counters_db.COUNTERS_DB, table_id, counter)
+                if counter_data is None:
+                    counts[table_id] = 0
+                else:
+                    counts[table_id] = int(counter_data)
+            return counts
+
+    def get_counts_table(self, counters, object_table):
+        """
+            Returns a dictionary containing a mapping from an object (like a port)
+            to its pg drop counts. Counts are contained in a dictionary that maps
+            counter oid to its counts.
+        """
+        counter_object_name_map = self.counters_db.get_all(self.counters_db.COUNTERS_DB, object_table)
+        current_stat_dict = OrderedDict()
+
+        if counter_object_name_map is None:
+            return current_stat_dict
+
+        for obj in natsorted(counter_object_name_map):
+            current_stat_dict[obj] = self.get_counts(counters, counter_object_name_map[obj])
+        return current_stat_dict
+
+
+    def clear_drop_counts(self):
+        """
+            Clears the current pg drop counter.
+        """
+
+        counter_pg_drop_array = [ "SAI_INGRESS_PRIORITY_GROUP_STAT_DROPPED_PACKETS"]
+        try:
+            pickle.dump(self.get_counts_table(
+                counter_pg_drop_array,
+                COUNTERS_PG_NAME_MAP),
+                open(self.port_drop_stats_file, 'wb+'))
+        except IOError as e:
+            print(e)
+            sys.exit(e.errno)
+        print("Cleared pg drop counter")
+
+def main():
+    parser = argparse.ArgumentParser(description='Display pg drop counter',
+                                     formatter_class=argparse.RawTextHelpFormatter,
+                                     epilog="""
+Examoles:
+pg_drop -c show
```

Comment:
do we usually use _ or - ? lets make sure we are aligned as after that it is difficult to change.